src/main/java/pvs/app/controller/ProjectController.java

```diff
@@ -133,4 +133,20 @@ public ProjectController(ProjectService projectService, RepositoryService reposi
         //         3        //
         //////////\\\\\\\\\\\\
     }
+
+    @PostMapping("/project/remove/{projectId}")
```

Comment:
From a general perspective, this API will be called to delete a specific resource, so it is recommended to use the DELETE request method.

```suggestion
    @DeleteMapping("/project/remove/{projectId}")
```

Reply:
Actually, this API is to modify a resource, to update the data, so I used PATCH in the beginning. However, there was something wrong with CORS. I have not found the solution.

Reply:
In my opinion, the front end doesn't need to know how the backend implements the delete feature, even it is just changing a flag.

For the front end, what will be changed after calling this API is "delete", so suggest using the DELETE method.

But the final choice is still up to you, hope this could help you.

Reply:
Are post, patch, delete, and put requests functionally different? I mean that is it okay to post by delete requests? I don't know what the difference of their function is.

Reply:
37c9405